src/products/index.ts

```diff
@@ -17,27 +18,52 @@ export type Transformer = (product: Product) => { data: object, variants: object
 @alias('products')
 @tag('gb-products', require('./index.html'))
 class Products {
-
   structure: Structure = this.config.structure;
+
+  props: Products.Props = {
+    storeSection: StoreSections.DEFAULT
+  };
+
   state: Products.State = {
     products: []
   };
   productTransformer: Transformer = ({ data, meta }: Product) =>
     ({ ...ProductTransformer.transformer(this.structure)(data), meta })
 
   init() {
```

Comment:
You might want to move init() above prodctTransformer as all your other methods are underneath

Reply:
Done.